tools/IceRpc.Protobuf.Tools/ProtocTask.cs

```diff
@@ -132,3 +140,17 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
     /// <inheritdoc/>
     protected override void LogToolCommand(string message) => Log.LogMessage(MessageImportance.Normal, message);
 }
+
+/// <summary>Converts a byte array to a hexadecimal string.</summary>
+public static class HexStringConverter
```

Comment:
It doesn't belong here, it should be a private or local method in OutputHashTask